thrift/src/main/thrift/atoms/recipe.thrift

```diff
@@ -32,25 +32,36 @@ struct Range {
 struct Ingredient {
   1: required string item
   2: optional string comment
-  3: optional double quantity
+  // 3: optional double quantity
```

Comment:
Are we removing we expect `quantity` to be stored in `quantityRange`?

Reply:
Yes, we thought `quantityRange` could accommodate both